Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:20.04 AS build
 RUN apt-get update && \
     apt-get -y upgrade && \
-    apt-get install -y wget
+    apt-get install -y --no-install-recommends wget ca-certificates
 WORKDIR /tmp
 RUN wget https://dl.google.com/go/go1.24.0.linux-amd64.tar.gz && \
     tar -xf go1.24.0.linux-amd64.tar.gz && \
```

Comment:
`ca-certificates` need to be installed before executing `wget https://dl.google.com/go/go1.24.0.linux-amd64.tar.gz`